content/en/tutorials/reproducible-machine-learning-cloud-pods/index.md

```diff
@@ -0,0 +1,230 @@
+
+{{< alert title="Saving Cloud Pods locally" >}}
+You can also export a Cloud Pod locally by specifying a file URI as an argument. To export on a local path, run the following command:
+
+{{< command >}}
+$ localstack pod save file://<path_on_disk>/<pod_name>
+{{< / command >}}
+
+The output of the above command will be a `<pod_name>.zip` file in the specified directory. We can restore it at any time with the `load` command.
+{{< /alert >}}
+
+To list all the Cloud Pods available locally or remote (LocalStack Pod's platform), you can use the `list` command:
+
+{{< command >}}
+$ localstack pod list
+
+localstack pod list
+┏━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ local/remote ┃ Name            ┃
+┡━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ local+remote │ reproducible-ml │
+└──────────────┴─────────────────┘
+{{< / command >}}
+
+You can inspect the contents of a Cloud Pod using the `inspect` command:
+
+{{< command >}}
+$ localstack pod inspect reproducible-ml
+{{< / command >}}
+
+While you save a Cloud Pod, it is automatically published on the LocalStack Pod's platform and can be shared with other users in your organization. While saving an already existing Cloud Pod, we would create a new version, which is eventually uploaded to the LocalStack Pod's platform.
+
+{{< alert title="Setting visibility of a Cloud Pod" >}}
+You can optionally set the visibility of a Cloud Pod to `private` or `public` using the `--visibility` flag. By default, the visibility of a Cloud Pod is set to `private`. To set a Cloud Pod to `public`, you can use the following command:
+{{< command >}}
+$ localstack pod push --name <pod_name> --visibility public
+{{< / command >}}
+The above command does not create a new version and requires a version already registered with the platform.
+{{< /alert >}}
+
+You can also attach an optional message and a list of services to a Cloud Pod using the `--message` and `--services` flags. You can check all the Cloud Pods in your organization over the [LocalStack Web Application](https://app.localstack.cloud/cloudpods). Now that we have created a Cloud Pod, we can ask one of our team members to start LocalStack and load the Cloud Pod using the `load` command.
+
+{{< command >}}
+$ localstack pod load reproducible-ml
+{{< / command >}}
+
+The `load` command will retrieve the content of our Cloud Pod named `reproducible-ml` from the LocalStack Pod's platform and inject it into our running LocalStack instance. Upon successfully loading the Cloud Pod, the Lambda function can be invoked again, and the log output should be the same as before.
+
+LocalStack Cloud Pods also feature different merge strategies to merge the state of a Cloud Pod with the current LocalStack instance. You can use the `--merge` flag to specify the merge strategy. The available merge strategies are:
+
+- **Inject with overwrite**: This is the default merge strategy. It will inject the state of the Cloud Pod into the current LocalStack instance and overwrite the existing state.
```

Comment:
Should we rename `inject` to load, since we changed the terminology in the CLI? If so, we also need to update the figure (I can take care of that eventually).

Reply:
Will update it 👍